tests/manifold/combination_manifold_01.cc

```diff
@@ -0,0 +1,75 @@
+//----------------------------  function_manifold_chart ---------------------------
+//    Copyright (C) 2011 - 2015 by the mathLab team.
+//
+//    This file is subject to LGPL and may not be  distributed
+//    without copyright and license information. Please refer
+//    to the file deal.II/doc/license.html for the  text  and
+//    further information on this license.
+//
+//---------------------------- function_manifold_chart ---------------------------
+
+
+// Test the combination of simple ChartManifolds: parabolic + translation
```

Comment:
can you rename the test and also mention ``CompositionManifold`` in the description here?